Add Zapstore project page (#629)
OpenSats/website: data/projects/zapstore.mdx

---
title: 'Zapstore'
dateAdded: '2025-03-27'
summary: 'A nostr-based app store for signed releases, open discovery, and censorship-resistant software distribution.'
nym: 'Franzap'
website: 'https://zapstore.dev/'
coverImage: '/static/images/projects/zapstore.png'
git: 'https://github.com/zapstore/zapstore'
tags: ['Nostr', 'Android', 'Mobile']
showcase: true
fund: nostr
announcementLink: '/blog/10th-wave-of-nostr-grants#zapstore'
---

> Zapstore is an app store built on nostr. Developers sign and publish releases as nostr events. Users can verify binaries before installation, discover apps through follows and recommendations, and install software outside the control of a single platform owner. The project supports Tor, local-first usage, and account-less flows.

Reviewer: ```suggestion
Zapstore is a mobile app store built on nostr for both Android and later for iOS. Developers sign and publish releases as nostr events. Users can verify binaries before installation, discover apps through follows and recommendations, and install software outside the control of a single platform owner. The project supports Tor, local-first usage, and account-less flows.
```

Author: I accepted this suggestion too quickly. Zapstore is not only for mobile apps. It is not purely a mobile app store.

Author: Good catch, reverted in bb2d79f5. The intro now reads:

> Zapstore is an app store built on nostr. ... The Android client is the most mature today, an iOS client is in development, and the same publishing flow works for desktop binaries and CLI tools.

Also adjusted the "Why fund it?" and "What's next?" sections so they don't frame the project as mobile-only.

Zapstore also ships developer tooling. `zapstore-cli` helps publish releases and manage app metadata, while the broader stack includes relays, an indexer, signing tools, and the Android client itself.

## Why fund it?

Software distribution is a choke point. Large app stores can delist apps, gate updates, and control how developers reach users. Zapstore gives developers a direct distribution path with signed releases and gives users another way to discover and install software they can verify.

> OpenSats supported Zapstore in the [tenth wave of nostr grants][wave-10]. The project helps build the publishing and discovery infrastructure that nostr apps need.

Reviewer: ```suggestion
OpenSats supported Zapstore in the [tenth wave of nostr grants][wave-10]. The project helps build the publishing and discovery infrastructure that Android and iOS apps need. 
```

## What's next?

> Recent work includes Zapstore 1.0 for Android, a redesigned package manager, custom app-catalog relays, curated app stacks, background update checks, and major improvements to the website and indexing pipeline. The indexer now supports a much larger app catalog, and the Android client has shipped steady 1.0.x releases.

Reviewer: ```suggestion
Recent work includes Zapstore 1.0 for Android, a redesigned package manager, custom app-catalog relays, curated app stacks, background update checks, and major improvements to the website and indexing pipeline. The indexer now supports a much larger app catalog, and the Android client has shipped steady 1.0.x releases. Zapstore is also designing an iOS version of the application, which bypasses the Apple's native app store and promises an even better user experience. 
```
